doctest/build.gradle

```diff
@@ -19,9 +19,12 @@ def path = project(':').projectDir
 // plugin module, and will only build ppl in it.
 def plugin_path = project(':doctest').projectDir
 
-task cloneSqlCli(type: Exec) {
+task cloneSqlCli() {
```

Comment:
@MitchellGale is going to fix this in https://github.com/Bit-Quill/opensearch-project-sql/pull/271
I think you can revert this to avoid conflicts

Reply:
thanks, reverted